src/test/java/org/lsc/plugins/connectors/james/TMailContactDstServiceTest.java

```diff
@@ -291,36 +288,6 @@ void shouldSupportPartialUpdate() throws Exception {
             CONTACT_RENE.getFirstname(), Optional.of("Surname")));
     }
 
-    @Test
-    void deleteShouldRemoveWhenTheContactExists() throws Exception {
-        createContact(CONTACT_RENE);
-
-        LscModifications modifications = new LscModifications(LscModificationType.DELETE_OBJECT);
-        modifications.setMainIdentifer(CONTACT_RENE.getEmailAddress());
-        modifications.setLscAttributeModifications(ImmutableList.of());
-
-        boolean applied = testee.apply(modifications);
-
-        assertThat(applied).isTrue();
-        // Wait for webadmin fix return 404 instead 200
-        assertThatThrownBy(() -> jamesDao.getContact(CONTACT_RENE.getEmailAddress()))
-            .isInstanceOf(NotFoundException.class);
-    }
-
-    @Test
-    void deleteShouldSucceedWhenTheContactDoesNotExist() throws Exception {
-        LscModifications modifications = new LscModifications(LscModificationType.DELETE_OBJECT);
-        modifications.setMainIdentifer(CONTACT_RENE.getEmailAddress());
-        modifications.setLscAttributeModifications(ImmutableList.of());
-
-        boolean applied = testee.apply(modifications);
-
-        assertThat(applied).isTrue();
-        // Wait for webadmin fix return 404 instead 200
-        assertThatThrownBy(() -> jamesDao.getContact(CONTACT_RENE.getEmailAddress()))
-            .isInstanceOf(NotFoundException.class);
-    }
-
```

Comment:
Why remove these tests? The service still have a delete method that deserves to be tested...

Reply:
Cause you suggested to disabled delete upon?

rf: https://github.com/linagora/tmail-lsc/pull/2#discussion_r856376107